spec/system/nsm/view_claim_spec.rb

```diff
@@ -308,6 +308,68 @@
       end
     end
 
+    context 'youth court fee' do
+      # rubocop:disable Style/HashSyntax
+      let(:claim) do
+        create(:claim, :case_type_magistrates,
+               :firm_details, :letters_calls,
+               work_items:, disbursements:,
+               plea:, plea_category:,
+               include_youth_court_fee:,
+               state: :submitted)
+      end
+      # rubocop:enable Style/HashSyntax
+
+      let(:plea) { :guilty }
+      let(:plea_category) { :category_1a }
+
+      describe 'show youth court CaseOutcomeCard' do
+        describe 'include_youth_court_fee true' do
+          let(:include_youth_court_fee) { true }
+
+          it 'show case outcome and additional fee' do
+            visit nsm_steps_view_claim_path(claim.id)
+
+            within('.govuk-summary-card', text: 'Case disposal') do
+              expect(page).to have_content('Category 1a')
+              expect(page).to have_content('Guilty')
+              expect(page).to have_content('Additional fee')
+              expect(page).to have_content('Youth court fee claimed')
+            end
+          end
+        end
+
+        describe 'include_youth_court_fee false' do
+          let(:include_youth_court_fee) { false }
+
+          it 'show case outcome and additional fee' do
+            visit nsm_steps_view_claim_path(claim.id)
+
+            within('.govuk-summary-card', text: 'Case disposal') do
+              expect(page).to have_content('Category 1a')
+              expect(page).to have_content('Guilty')
+              expect(page).to have_content('Additional fee')
+              expect(page).to have_content('Youth court fee not claimed')
+            end
+          end
+        end
+      end
+
+      describe 'include_youth_court_fee is nil' do
+        let(:include_youth_court_fee) { nil }
+
+        it 'show pre-yout court changes case disposal view' do
+          visit nsm_steps_view_claim_path(claim.id)
+
+          within('.govuk-summary-card', text: 'Case disposal') do
+            expect(page).to have_content('Category')
+            expect(page).to have_content('Guilty plea')
+            expect(page).not_to have_content('Additional fee')
+          end
+        end
```

Comment:
This might indicate updates are needed for your code --> There are examples where youth court included = nil but you still need to show the new version of the case category card (without the additional fee row) because you need to present the "Other" case outcome with the custom text which is untested for

Reply:
yeah I think the way you did it _generally_ works (if we are able to get the work done on time), but it does miss out some Cat B scenarios